### PR TITLE
added clarity on python error raising

### PR DESCRIPTION
**Pull request by Andy-Nkumane**
Related issues: [please specify]

## Description:
specified error to be used so that it matched the perfect project for the automarker

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by TheAbstract — COMMENTED** [review]
@Andy-Nkumane @Samantha-Hampton I'm not sure about adding this much specificity to the instructions, it's kinda telling the learners what they should learn. For example if someone raises a RuntimeError here (which I've seen) then clearly this person doest know what this error actually is, and they have some learning to do. Now if we just tell them what error to raise, this learning doest actually happen. Most of our learners will raise the error we tell them to, without understanding why it should be this one and not some other type of error.

**Comment by Samantha-Hampton** [comment]
@Andy-Nkumane and @TheAbstract 
Siphiwe, I think I agree with you, but thinking that maybe we can add a resource link to help them with how they choose the correct error type to use (or point them back to place in curriculum where it is taught)? So they know that they can't just willy-nilly choose any error type they fancy but not spoon feed them either?

**Comment by ry-oc** [comment]
@Andy-Nkumane please resolve the merge conflicts

File: content/projects/tdd/password-checker/part1/_index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ passwordIsValid(password)
 password_is_valid(password)
 ```
 
-`password_is_valid` will check if the password meets a few different conditions. If one of the below conditions is not met, then the relevant error/exception should be thrown/raised. Your error/exception message should match one of the following conditions exactly (word-for-word).
+`password_is_valid` will check if the password meets a few different conditions. If one of the below conditions is not met, then the relevant error/exception should be thrown/raised. Python should raise a specific error type according to this, [here](https://www.tutorialsteacher.com/python/error-types-in-python) are the types of error that we get in Python. Your error/exception message should match one of the following conditions exactly (word-for-word).
```
Comment (Samantha-Hampton) [inline content/projects/tdd/password-checker/part1/_index.md]: Just for clarity and slight grammatical fix, I'd say:

Python should raise a specific error type according to this. [This resource] (https://www.tutorialsteacher.com/python/error-types-in-python) shows the types of errors that we get in Python, select the appropriate one for this application.

**Review by Samantha-Hampton — CHANGES_REQUESTED** [review]
Minor change for clarity

**Review by TheAbstract — APPROVED** [review]

**Review by Samantha-Hampton — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**